src/Linux/Host/Broker/Impl/Services/LinuxRHostProcessService.cs

```diff
@@ -41,15 +45,37 @@ class LinuxRHostProcessService : IRHostProcessService {
             return new UnixProcess(process);
         }
 
+        private string GetUsrLibPath() {
+            const string x64Lib = "/usr/lib/x86_64-linux-gnu";
```

Comment:
when i was testing from VS Code this was the value for `PATH=
/home/user1/bin:/home/user1/.local/bin:/bin:/usr/bin:/usr/X11R6/bin:/usr/local/bin:/snap/bin`. I was trying to only include paths that exist. I don't think this is needed. Will remove.

For, `LD_LIBRARY_PATH', I wanted to make sure it works if a user decides to start broker from a command line. Again not absolutely needed.